README.md

```diff
@@ -86,6 +86,8 @@ lang | Sets the language.
 chartControlsWidgets | Render function for chart control widgets. Refer to [Customising Components](#customising-components).
 topWidgets | Render function for top widgets. Refer to [Customising Components](#customising-components).
 theme | Sets the chart theme. themes are (`dark\|light`), and default is `light`.
+symbol | Sets the current active symbol.
+isMobile | Switch between mobile or desktop view. Defaults to `true`.
```

Comment:
isMobile should default to `false`.

Reply:
When binary-static and binary-bot are using this as a library, they will use it as `isMobile` is `true`. For library users this would be the expected setting, unless they want to use the chart as a full app.